chemie/customprofile/migrations/0006_profile_voted.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Was this migration committed by accident? This could create some conflicts with the master repo.

Reply:
This should be looked into before merge

Reply:
This migration is correct. Without it, users cannot vote.

Reply:
The migration caused a NodeNotFoundError for me. However, upon removing it and running makemigrations everything seemed to work fine. Are you sure it should be in the remote branch when merging?